### DISCUSSION TITLE
Keys section issue

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#6093] [Issues]
Hi,
The keys sections have some issue on my server.
 I can't remove the entries and are represented as blank keys.
Thanks,

![Keys](https://github.com/Ylianst/MeshCentral/assets/102313998/e83aa69e-3820-4bed-81d5-5017f14d9ed7)

**Comment by si458** [comment]
are you using minify? try turning it off if so
also have u customised the default.handlebars and then translated it at all?

**Reply by ghost** [reply]
Yes I did a translation and how and I don't have any minify in the config file
Thanks.

**Reply by ghost** [reply]
What happens if I backup the server and reinstall? WIll the problem persists due to backup?

**Reply by si458** [reply]
No idea, u won't know until u try it!

**Reply by ghost** [reply]
OK so reinstalling didn't help.
As I thought, the backup includes the issue.

**Reply by ghost** [reply]
I guess I need to install from scratch.
I don't see any fix for it unless you have another idea.
